pkg/stack/stack_test.go

```diff
@@ -8,7 +8,7 @@ import (
 func Test_applyTemplateWithAuth(t *testing.T) {
 
```

Comment:
Would be nice to see this in a const within the method or a string. Good improvement 👍